akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -106,7 +106,7 @@ class CircuitBreaker(
   scheduler:                Scheduler,
   maxFailures:              Int,
   callTimeout:              FiniteDuration,
-  resetTimeout:             FiniteDuration,
+  val resetTimeout:         FiniteDuration,
```

Comment:
The problem I was thinking of is that you can't add a new method to a class that could have subclass implementation because the subclass might have already implemented such method and the signatures might not match (e.g. different return types). I guess that is a source compatibility issue and not a binary compatibility issue. I think it's fine here, since this class is not intended to be subclassed in the first place.